pkg/native/unserializers/unserializer_cdx.go

```diff
@@ -37,6 +65,24 @@ func (u *UnserializerCDX14) ParseStream(_ *options.Options, r io.Reader) (*sbom.
 		NodeList: &sbom.NodeList{},
 	}
 
+	metadata := bom.Metadata
+	if metadata.Lifecycles != nil {
+		for _, lc := range *metadata.Lifecycles {
+			lc := lc
+			name := lc.Name
+			desc := lc.Description
+			t := u.phaseToSBOMType(&lc.Phase)
+			if name == "" {
+				name = string(lc.Phase)
+			}
+
+			md.DocumentTypes = append(md.DocumentTypes, &sbom.DocumentType{
+				Name:        &name,
+				Description: &desc,
+				Type:        t,
+			})
+		}
+	}
```

Comment:
This would be removed once the other PR is ready and we rebase (and fix conflicts), please ignore